orchagent/portsorch.cpp

```diff
@@ -6017,15 +6104,27 @@ void PortsOrch::generateQueueMapPerPort(const Port& port, FlexCounterQueueStates
         }
 
         queueVector.emplace_back(name.str(), id);
-        queuePortVector.emplace_back(id, sai_serialize_object_id(port.m_port_id));
+        if (voq)
+        {
+            queuePortVector.emplace_back(id, sai_serialize_object_id(port.m_system_port_oid));
+        }
+        else
+        {
+            queuePortVector.emplace_back(id, sai_serialize_object_id(port.m_port_id));
+        }
 
```

Comment:
![image](https://user-images.githubusercontent.com/46945843/211689877-b021f1ed-38aa-4a09-bf87-1442c524b1c0.png)

this code seems to be present in master but not here. is this needed ?

Reply:
Did you mean the setting of queueTypeVector and queueIndexVector?

Reply:
This is existing code and not added by the pr. If you expand the diff you will see the code in 202205 as well.